packages/sdks/package.json

```diff
@@ -63,7 +63,7 @@
     "upgrade-example:all": "yarn loop upgrade-example latest"
   },
   "dependencies": {
-    "@builder.io/mitosis": "^0.3.8",
+    "@builder.io/mitosis": "latest",
```

Comment:
I prefer a fixed or carat version instead of `latest` here. I know we maintain mitosis, but i worry that `latest` might cause more frequent updates to mitosis than necessary.

Reply:
yeah that was a mistake i just caught as well, `ni @builder.io/mitosis@latest` i always forget does this for somereason (whereas using yarn/npm directly doens't)